src/apps/auth-login/LoginMethodSelector/LoginMethodSelector.tsx

```diff
@@ -0,0 +1,73 @@
+import { ReactNode, forwardRef, useState } from "react";
```

Comment:
There are some unused imports here: `useState`, `supabase` and `updateUserProjectGroup`.

Reply:
Fixed